CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [0.2.1] - 2024-04-03
+
+- Add routing to `LSP.executeCommand` server runtimes implementation: not runtimes will route LSP commands for execution to server between multiple registered servers
```

Comment:
nit: typo "not runtimes" 

Can we add a bit more info about how this routing is done also?